packages/web/src/shared/navigation.tsx

```diff
@@ -4,26 +4,35 @@ import Button, { BTN } from 'src/shared/Button.3'
 import OvalCoin from 'src/shared/OvalCoin'
 import { colors, textStyles } from 'src/styles'
 
+export enum NavigationTheme {
+  LIGHT,
+  DARK,
+}
+
 interface Props {
   link?: string
   onPress?: () => void
   text: string
   selected: boolean
+  theme?: NavigationTheme
```

Comment:
btw I think from a design perspective there is no reason that you couldnt have gold and light or green on dark. 

but we can always tackle that if time comes.

Reply:
I did in this way because is necessary to set the button theme too